pom.xml

```diff
@@ -319,6 +319,32 @@
     <!-- Dependencies -->
     <dependencies>
         <!-- Servlets 2.3 - JSTL 1.1 -->
+            <dependency>
+	<groupId>com.fasterxml.jackson.core</groupId>
+	<artifactId>jackson-core</artifactId>
+	<version>2.5.4</version>
+</dependency>
+<dependency>
+	<groupId>com.fasterxml.jackson.core</groupId>
+	<artifactId>jackson-databind</artifactId>
+	<version>2.5.4</version>
+</dependency>
+<dependency>
+	<groupId>com.fasterxml.jackson.core</groupId>
+	<artifactId>jackson-annotations</artifactId>
+	<version>2.5.4</version>
+</dependency>
+<dependency>
+	<groupId>com.fasterxml.jackson.dataformat</groupId>
+	<artifactId>jackson-dataformat-yaml</artifactId>
+	<version>2.5.4</version>
+</dependency>
+
+		<dependency>
+			<groupId>com.amazonaws</groupId>
+			<artifactId>aws-java-sdk-ec2</artifactId>
+			<version>1.10.37</version>
+		</dependency>
         <dependency>
```

Comment:
Can you seperate out the dependency changes into a new PR (did they get committed by mistake?)